tests/E2E Tests/WebAppUiTests/TestingWebAppLocally.cs

```diff
@@ -48,14 +48,16 @@ public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPassword()
         await ExecuteWebAppCallsGraphFlow(labResponse.User.Upn, labResponse.User.GetOrFetchPassword(), clientEnvVars, TraceFileClassName).ConfigureAwait(false);
     }
 
-    [Fact]
+    [Theory]
+    [InlineData("https://MSIDLABCIAM6.ciamlogin.com")] //Non CIAM CUD authority
```

Comment:
I think we can just say `// CIAM authority` here and in the next test, otherwise it gets confusing.

Reply:
Updated